tfx/components/example_validator/executor_test.py

```diff
@@ -46,26 +46,82 @@ def testDo(self):
         self._testMethodName)
 
     validation_output = standard_artifacts.ExampleAnomalies()
-    validation_output.uri = os.path.join(output_data_dir, 'output')
+    validation_output.uri = output_data_dir
+    validation_output.split_names = artifact_utils.encode_split_names(
+        ['eval'])
 
     input_dict = {
         executor.STATISTICS_KEY: [eval_stats_artifact],
         executor.SCHEMA_KEY: [schema_artifact],
     }
+
+    exec_properties = {
+        executor.EXCLUDE_SPLITS_KEY: None
+    }
+
     output_dict = {
         executor.ANOMALIES_KEY: [validation_output],
     }
 
-    exec_properties = {}
+    example_validator_executor = executor.Executor()
+    example_validator_executor.Do(input_dict, output_dict, exec_properties)
+    self.assertEqual(['anomalies.pbtxt'],
+                     tf.io.gfile.listdir(os.path.join(validation_output.uri,
+                                                      'eval')))
+    anomalies = io_utils.parse_pbtxt_file(
+        os.path.join(validation_output.uri, 'eval', 'anomalies.pbtxt'),
+        anomalies_pb2.Anomalies())
+    self.assertNotEqual(0, len(anomalies.anomaly_info))
+    # TODO(zhitaoli): Add comparison to expected anomolies.
+
+  def testDoWithExcludedSplits(self):
```

Comment:
you can just add it to testDo, it's not two different branch in the code as ExcludedSplit is in your main code branch